app/apollo/models/organization.local.schema.js

```diff
@@ -45,6 +45,19 @@ const OrganizationLocalSchema = new mongoose.Schema({
   },
 });
 
+OrganizationLocalSchema.statics.getRegistrationUrl = async function(org_id, context) {
+  context.logger.debug({org_id}, 'getRegistrationUrl enter');
+  const org = await this.findById(org_id);
+  if (process.env.EXTERNAL_URL) {
+    return {
+      url: `${process.env.EXTERNAL_URL}/api/install/razeedeploy-job?orgKey=${org.orgKeys[0]}`,
+    };
+  }
+  return {
+    url: `http://localhost:3333/api/install/razeedeploy-job?orgKey=${org.orgKeys[0]}`,
```

Comment:
Is there a way to get the real address instead of hardcoding it to localhost?

Reply:
fixed and removed EXTERNAL_URL env.